src/Compilers/VisualBasic/Test/Semantic/Compilation/SemanticModelAPITests.vb

```diff
@@ -4274,5 +4274,31 @@ End Class
         End Sub
 #End Region
 
+        <Fact, WorkItem(1146124, "DevDiv")>
+        Public Sub GetTypeInfoForXmlStringInCref()
+            Dim compilation = CompilationUtils.CreateCompilationWithMscorlibAndVBRuntime(
+<compilation name="GetSemanticInfo">
+    <file name="a.vb"><![CDATA[
+Module Program
+    ''' <summary>
+    ''' <see cref=""/>
+    ''' </summary>
+    ''' <param name="args"></param>
+    Sub Main(args As String())
+
+    End Sub
+End Module
+    ]]></file>
+</compilation>)
+
+            Dim tree As SyntaxTree = (From t In compilation.SyntaxTrees Where t.FilePath = "a.vb").Single()
+            Dim root = tree.GetCompilationUnitRoot
+            Dim xmlString = root.DescendantNodes(descendIntoTrivia:=True).OfType(Of XmlStringSyntax).Single()
```

Comment:
It isn't clear to me which node is being selected here. Would it be more clear to select the node by its ToString?